README.md

```diff
@@ -1 +1,130 @@
- # cvs-tsk-update-store
+# cvs-tsk-update-store
+
+## Overview
+Lambda handler and business logic for the CVS task "Update Store":
+
+1. Receives DynamoDB stream events, containing NoSQL document snapshots ("**images**")
+2. Converts these images to a TypeScript model, then to an SQL-friendly tuple.
+3. Upserts (inserts or updates) tuples into RDS [Aurora][aurora], a relational SQL database.
+
```

Comment:
Will probably need your help to run the `test-local-i` script locally please.
I can't also get the `start` script to work.
I think some dependencies are missing for me and fail to make this script run when you have time.

Reply:
Resolved as per our chat yesterday